src/functions-and-arrays.js

```diff
@@ -1,18 +1,80 @@
 // Iteration #1: Find the maximum
+var maxOfTwoNumbers = function(num1, num2){
+  if (num1 > num2){
+    return num1;
+  } else {
+    return num2;
+  }
+}
+
```

Comment:
Good job! Clean and easy to understand! 👍

Reply:
Thank you, Marta!